src/useDebouncedCallback.ts

```diff
@@ -32,8 +32,7 @@ export default function useDebouncedCallback<T extends (...args: any[]) => Retur
   funcRef.current = func;
 
   // Bypass `requestAnimationFrame` by explicitly setting `wait=0`.
-  const useRAF =
-    !wait && wait !== 0 && typeof window !== 'undefined' && typeof window.requestAnimationFrame === 'function';
+  const useRAF = !wait && wait !== 0 && typeof window !== 'undefined';
```

Comment:
Why have I removed `typeof window.requestAnimationFrame === 'function'`? 

1. RAF is supported by 98% browsers including IE11.

2. (more important) React itself doesn't work in browsers with no `requestAnimationFrame` support.

It means that there is no sense to add this condition because if the browser doesn't support RAF and there is no polyfill, React won't work. If React doesn't work, it cant run your hook. So we can be 100% sure if your hook was called, `window.requestAnimationFrame` is present.

See:
https://reactjs.org/docs/javascript-environment-requirements.html

![image](https://user-images.githubusercontent.com/206567/97196121-2875d200-17bd-11eb-8310-680113a24df2.png)

Reply:
Oh, I see, great!